examples/2d/regularized_inverse_scattering_MNIST_torch.py

```diff
@@ -84,7 +86,7 @@ def forward(self, input_tensor):
 
     transforms_to_apply = transforms.Compose([
         transforms.ToTensor(),
-        transforms.Normalize((0.5, 0.5, 0.5), (0.5, 0.5, 0.5))  # Pixel values should be in [-1,1]
+        transforms.Normalize((0.5,), (0.5,))  # Pixel values should be in [-1,1]
```

Comment:
Don't see how this comment makes any sense. It normalizes the input to have mean 0.5 and standard deviation 0.5, no?

Reply:
Is it better?

Reply:
(I've simply removed this comment and explained it was simply a standard practice and for compatbility issues)